### DISCUSSION TITLE
How does the library display the frontend files from the `build` directory on production?

### DISCUSSION THREAD

**Discussion by YairLevi** [wailsapp/wails#3201] [General]
Hey all!
I have a question I'm hoping to get an answer to.
When wails builds the frontend app, say in react, and now creates an `.exe` file, I'm assuming it somehow binds the finished resources onto the window. I'm using some other library that uses the tiny-webview lib, but in Java. I was wondering if there is a way for me to replicate or at least get some inspiration on how to embed frontend build resources in the window. So far the only options for me were either load a url of some web server that serves those files, which works but is somewhat of a pain and not really elegant IMO. the other solution was to encode the raw html into the window, which I need to also do with the JS and CSS scripts, because otherwise it doesnt know the link them using <script>  or <link> tags.

 would love to get more info or tips on how to approach this.

**Comment by airtonix** [comment]
@YairLevi 

It uses https://pkg.go.dev/embed

Here: https://github.com/AlienRecall/wails-react-template/blob/main/main.tmpl.go#L15-L19
